### DISCUSSION TITLE
How to get a list of custom repository properties?

### DISCUSSION THREAD

**Discussion by heitorPB** [PyGithub/PyGithub#2993] [Q&A] [answered]
What is the equivalent of `https://api.github.com/repos/OWNER/REPO/properties/values`? I'm interested in getting/setting custom properties for my repos. I can do that with `curl`, but couldn't find a way to do it via this project.

Docs here: [Get all custom property values for a repository](https://docs.github.com/en/rest/repos/custom-properties?apiVersion=2022-11-28#get-all-custom-property-values-for-a-repository).

**Comment by dawngerpony** [comment, marked as answer]
[This comment](https://github.com/PyGithub/PyGithub/issues/2895#issue-2118964108) contains the answer to your question:

```python
my_repo.raw_data["custom_properties"]
```

#2968 appears to add proper support for custom properties, but doesn't look like it's made it into a release yet.

**Comment by EnricoMi** [comment]
#2968 has just been released as v2.4.0.
